pkg/cache/clusterqueue_test.go

```diff
@@ -0,0 +1,68 @@
+package cache
+
+import (
+	"testing"
+
+	kueue "sigs.k8s.io/kueue/apis/kueue/v1beta1"
+	"sigs.k8s.io/kueue/pkg/metrics"
+	utiltesting "sigs.k8s.io/kueue/pkg/util/testing"
+)
+
+func TestClusterQueueUpdateWithFlavors(t *testing.T) {
```

Comment:
I found that this is the only test we have for exported function that belongs to ClusterQueue :(
I will add more unit tests for that.

Reply:
Most of the things are tested as part of the Cache operations... but yeah, it could be useful